dev/user.clj

```diff
@@ -67,16 +69,26 @@
 
 (defn start
   "Starts CTIA with given config and services, otherwise defaults
-  to the same configuration as #'init/start-ctia."
+  to the same configuration as #'init/start-ctia (ES5)."
   [& {:keys [config services] :as m}]
-  (serially-alter-app 
+  (serially-alter-app
     (fn [app]
       (println "Starting CTIA...")
       (if app
         (do (println "CTIA already started! Use (go ...) to restart")
             app)
         (init/start-ctia! m)))))
 
+(defn -es7-init-config []
+  (-> (p/build-init-config)
+      (assoc-in [:ctia :store :es :default :version] 7)
+      (assoc-in [:ctia :store :es :default :port] 9207)))
+
+(defn start7
+  "Start CTIA with ES7"
+  []
+  (start :config (-es7-init-config)))
```

Comment:
I think `start` and `start7` should have the same interface to work as a replacement for each other. WDYT?

```suggestion
(defn start7
  "Start CTIA with ES7"
  [& {:as opts}]
  (apply start (update opts :config merge (-es7-init-config))))
```

this ^ suggestion is just to illustrate the approach. Looks like `merge` is not enough

Reply:
I made `-es7-init-config` a function because there's isn't a good answer here IMO.